src/Tools/Microsoft.dotnet-openapi/test/OpenApiTestBase.cs

```diff
@@ -143,33 +138,27 @@ public Task<IHttpResponseMessageWrapper> GetResponseAsync(string url)
             MemoryStream stream = null;
             if(result != null)
             {
-                byte[] byteArray = Encoding.ASCII.GetBytes(result.Item1);
+                var byteArray = Encoding.ASCII.GetBytes(result.Item1);
                 stream = new MemoryStream(byteArray);
             }
 
             return Task.FromResult<IHttpResponseMessageWrapper>(new TestHttpResponseMessageWrapper(stream, result?.Item2));
         }
     }
 
-    public class TestHttpResponseMessageWrapper : IHttpResponseMessageWrapper
+    public sealed class TestHttpResponseMessageWrapper : IHttpResponseMessageWrapper
     {
         public Task<Stream> Stream { get; }
 
         public HttpStatusCode StatusCode { get; } = HttpStatusCode.OK;
 
         public bool IsSuccessCode()
         {
-            switch(StatusCode)
+            return StatusCode switch
             {
-                case HttpStatusCode.OK:
-                case HttpStatusCode.Created:
-                case HttpStatusCode.NoContent:
-                case HttpStatusCode.Accepted:
-                    return true;
-                case HttpStatusCode.NotFound:
-                default:
-                    return false;
-            }
+                HttpStatusCode.OK or HttpStatusCode.Created or HttpStatusCode.NoContent or HttpStatusCode.Accepted => true,
+                _ => false,
+            };
```

Comment:
While I rarely remember how to _write_ a `switch` expression, they're certainly easy to read 😺